test/alphabet/alphabet_test.cpp

```diff
@@ -115,7 +115,7 @@ TYPED_TEST(alphabet, to_rank)
 
 TYPED_TEST(alphabet, copy_constructor)
 {
-    constexpr underlying_rank_t<TypeParam> rank = (alphabet_size_v<TypeParam> == 1) ? 0 : 1;
+    constexpr underlying_rank_t<TypeParam> rank = 1 % alphabet_size_v<TypeParam>;
```

Comment:
Can you document this method once at the top, since it is used throughout the file and it is less obvious what it does compared to the previous solution?